x-pack/plugins/endpoint/public/applications/endpoint/store/endpoint_list/reducer.ts

```diff
@@ -16,7 +18,10 @@ const initialState = (): EndpointListState => {
   };
 };
 
-export const endpointListReducer = (state = initialState(), action: EndpointListAction) => {
+export const endpointListReducer: Reducer<EndpointListState, AppAction> = (
```

Comment:
Q. Why `AppAction` and not `EndpointListAction`? 
is it so that in the future we could take advantage of other concern's Actions?

Reply:
its because `combineReducers` passes in all actions, not just `EndpointListAction`

Reply:
Ah Ok - but were you getting TS errors?

I think the benefit was that types shown here would be narrowed down to only the ones this concern cares about.

Reply:
We weren't getting errors but I think to your previous point it might be good to take advantage of being able to use other concern's or generic/global app actions in the future and this would keep that ability. Kind of like how we have the page navigated actions now. I do see what you're saying though, i don't necessarily disagree with it

Reply:
👍

Reply:
even if we filter out non-concern specific actions in the future, they aren't yet filtered out, so this type is more correct